### DISCUSSION TITLE
Budi-CLI on ARM-based server

### DISCUSSION THREAD

**Discussion by manwithpowers** [Budibase/budibase#10509] [Help]
I've tried installing the Budi-CLI to manage my existing Budibase installation but am getting an `/usr/local/bin/budi: cannot execute binary file: Exec format error` when trying to run it (eg `budi help`). I'm running an Ampere (ARM) Oracle Cloud Instance and it looks like that could be the issue - does anyone know if it's possible to run the CLI on ARM architecture? The docs seem to imply it's possible since they reference editing the `docker-compose.yaml` file.
Thanks in advance!

**Comment by AlejandroFNadal** [comment]
I have run it on a raspi recently. I needed to use budi hosting --init --single or budi hosting --init single, or sth like that. The docs suggestion to edit the docker-compose did not work for the system composed of multiple dockers.

**Reply by manwithpowers** [reply]
Thanks, I already have Budibase running in a single docker container, I just wanted to use the CLI to run a backup and am getting the error trying to do anything with `budi`
